src/main/scala/firrtl/annotations/Target.scala

```diff
@@ -0,0 +1,342 @@
+// See LICENSE for license details.
+
+package firrtl
+package annotations
+
+import firrtl.ir.Expression
+import AnnotationUtils.{toExp, validComponentName, validModuleName}
+import TargetToken._
+
+import scala.collection.mutable
+
+
+/**
+  * Refers to something in a FIRRTL [[firrtl.ir.Circuit]]. Used for Annotation targets.
```

Comment:
This is a fairly minor thing, and ScalaDoc does not mandate a style, but I think it is more common to have the explanation start on the first line of the comment, ie.
```scala
/** Here is a short description
   * === This line often left blank ===
   * Here starts longer descriptions
   * ...
   */
```
Starting on the second line follows JavaDoc style, and for that reason is probably the default in IntelliJ

Reply:
Is the `*` of subsequent lines supposed to be lined up with the second `*` on the first line?